client/js/components/AddRecipePage.jsx

```diff
@@ -119,30 +114,42 @@ export class AddRecipePage extends React.Component {
     }
   }
   /**
-   *
    *
    * @memberof AddRecipePage
+   *
    * @returns {null} null
    */
   handleSelectImage() {
     $('input[type=file]').click();
```

Comment:
Why make use of `jquery` here? Can't you pass an onClick method to the input handling this?